tests/utils/test_events.py

```diff
@@ -360,7 +360,7 @@ def test_NthTradingDayOfWeek(self, n):
     @subtest(param_range(MAX_WEEK_RANGE), 'n')
     def test_NDaysBeforeLastTradingDayOfWeek(self, n):
         cal = get_calendar('NYSE')
-        rule = NDaysBeforeLastTradingDayOfWeek(n)
+        rule = NDaysBeforeLastTradingDayOfWeek(n, True)
```

Comment:
Yeah, that makes sense. But you had suggested that the subclasses should share an `__init__`